pkg/performanceprofile/controller/performanceprofile/hypershift/hypershift.go

```diff
@@ -4,9 +4,12 @@ import (
 	"bytes"
```

Comment:
this should have been 2 commits, 1 to add List, 1 to add Update. OK for now, let's take into account next time.

Reply:
Moved into a separate PR: https://github.com/openshift/cluster-node-tuning-operator/pull/1107